### PR TITLE
Avoid the spot lights flickering in Home

### PR DESCRIPTION
**Pull request by samcake**
- THe flickering comes from the transition between the Inside / outside volume of the spot light. We use 2 different techniques depending of the region in which is the eye to render the spot light.
in master, this transition fails when looking at the spot light origin from under (in the cone), we then are in a zone which is  OUTSIDE but the technique used for outside is failing there.

- To fix this quickly, i simply extended the zone considered  "INSIDE", this avoid to go through the problematic region.

- REfreshed some of the tools we use for monitoring rendering and put them in utilities / render to avoid one more folder level (they were in utilities/tools/render)

TO TEST:
Go in forum at the entrance of the dressing room.
Make sure the spot lights are turned on (switch against the wall to your left)
now translate toward the wall back and forth.
In master, you should see the spot lights flickering
not with this build

### CODE REVIEW

File: libraries/render-utils/src/DeferredLightingEffect.cpp [hunk 1]
```diff
@@ -435,8 +435,9 @@ void DeferredLightingEffect::render(const render::RenderContextPointer& renderCo
                     float expandedRadius = light->getMaximumRadius() * (1.0f + SCALE_EXPANSION);
                     // TODO: We shouldn;t have to do that test and use a different volume geometry for when inside the vlight volume,
                     // we should be able to draw thre same geometry use DepthClamp but for unknown reason it's s not working...
+                    const float OVER_CONSERVATIVE_SCALE = 1.1;
```
Comment (samcake) [inline libraries/render-utils/src/DeferredLightingEffect.cpp]: This is the fix, scaling the Radius of the cone by 10%

**Comment by hifi-gustavo** [comment]
Build console output for commit: b492c4b52c31c5cf61885486ee0c9a4687db8b6f available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by kevinmthomas-carpool** [comment]
Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe does not exist.  Can you repopulate.

**Comment by samcake** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11b6952eff659fc91dd13c610c04e213a0796d44 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11b6952eff659fc91dd13c610c04e213a0796d44 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by kevinmthomas-carpool** [comment]
hi @samcake the windows Installer is not there.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0ee1f039e9ba0dd52825e3fd3b2809aba8307cf6 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by samcake** [comment]
I don;t see any real issue, looks like a jenkins build issue

**Comment by samcake** [comment]
test this please

**Comment by kevinmthomas-carpool** [comment]
I don't have any way of grabbing the Windows binary.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0ee1f039e9ba0dd52825e3fd3b2809aba8307cf6 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by kevinmthomas-carpool** [comment]
Success.  Tested first with current Windows Build 4570 and when entering the dressing room in forum when I toggled the lights and I moved my avatar the lights flashed.  Testing with PR7509 this was not the case and the lights worked normally.

**Comment by samcake** [comment]
PLease verify that the "renderStats.js" works correclty
you need to use the one provided in this PR
so link to the file:

**Comment by samcake** [comment]
PLease verify that the "renderStats.js" works correclty
you need to use the one provided in this PR
so link to the file:
https://raw.githubusercontent.com/samcake/hifi/orange/examples/utilities/render/renderStats.js

**Comment by kevinmthomas-carpool** [comment]
@samcake verified https://raw.githubusercontent.com/samcake/hifi/orange/examples/utilities/render/renderStats.js
works correctly.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3cf2eeb4d57e977a417c394d273255c82cc50321 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f09593e10627f6bb2db9fce68c96a6df6d76ae55 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6213e0859d357bc11e99ff6fb7037fb35a5f15c2 available at: https://gist.github.com/b805140aa0daf25ad25ae8ee60af29a7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7509.dmg

**Pull request merged**